pkg/secrets/secrets.go

```diff
@@ -127,9 +126,33 @@ func getMountsMap(path string) (string, string, error) {
 	return "", "", errors.Errorf("unable to get host and container dir")
 }
 
-// SecretMounts copies the contents of host directory to container directory
+// SecretMounts copies, adds, and mounts the secrets to the container root filesystem
+func SecretMounts(mountLabel, containerWorkingDir string) ([]rspec.Mount, error) {
```

Comment:
This function never returns an error. I think it's probably worth returning the errors we get back if we fail to mount something, though failures should not be fatal.

Reply:
Correction - it looks like adding the FIPS mode secret causes an error, but the default and override mounts files don't. Can we store errors from those two, so we don't fail immediately if one of them fails but we still return an error at the end?

Reply:
It returns an error when it fails to add the FIPS mode secret if /etc/system-fips exists on the host. The other errors end up being logged.

Reply:
fixed